pep-0695.rst

```diff
@@ -875,6 +878,181 @@ incorrectly, because functions within a class do not normally have access to
 the enclosing class namespace. The implementation will retain a reference to
 the class namespace so that class-scoped names are resolved correctly.
 
+.. _695-scoping-behavior:
+
+Scoping Behavior
+----------------
+
+The new syntax requires a new kind of scope that behaves differently
```

Comment:
I think I can follow this explanation, but I'd happier if the examples used the new scope without trying to make it look like a function.

A function definition defines a scope, but it also is a statement that creates a function.
The new scopes do not create functions (the implementation might, but there is no requirement for them to do so).

I think the comparison to functions is misleading.

Reply:
In what way is it misleading? The new scope behaves like a function scope in terms of the behavior of cell variables, nonlocals, etc., so the comparison to functions provides a convenient way to inherit from that behavior. We could use an alternative syntax (like your `let` suggestion), but I believe it would make the equivalence harder to understand, as we'd have to explain more about exactly how `let` works.